SonarQube.Bootstrapper/BuildAgentUpdater.cs

```diff
@@ -99,6 +117,81 @@ public bool CheckBootstrapperApiVersion(string versionFilePath, Version bootstra
             return false;
         }
 
+
+        public void InjectLoaderTargets(ILogger logger)
+        {
+            WarnOnGlobalTargetsFile(logger);
+            InternalCopyTargetsFile(logger);
+        }
+
+        private static void InternalCopyTargetsFile(ILogger logger)
+        {
+            string sourceTargetsPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), LoaderTargetsName);
+            Debug.Assert(File.Exists(sourceTargetsPath), String.Format("Could not find the {0} file in the current director", LoaderTargetsName));
```

Comment:
Typo - "director".
The message is misleading: it isn't looking in the current directory, it's looking in the same directory as the executing assembly.